target/riscv/cpu.h

```diff
@@ -556,20 +556,25 @@ void riscv_cpu_register_gdb_regs_for_features(CPUState *cs);
 typedef CPURISCVState CPUArchState;
 typedef RISCVCPU ArchCPU;
 
-#include "exec/cpu-all.h"
-#include "cpu_cheri.h"
-
 #define TB_FLAGS_MMU_MASK   3
 // For capmode we pick any flags bit that isn't used yet, 0x100 right now
-#define TB_FLAGS_CAPMODE 0x100
+#define TB_FLAG_CHERI_CAPMODE 0x100
+#define TB_FLAG_CHERI_PCC_VALID 0x200 /* CHERI PCC is tagged, executable and unsealed */
 #define TB_FLAGS_MSTATUS_FS MSTATUS_FS
-_Static_assert((TB_FLAGS_CAPMODE & TB_FLAGS_MSTATUS_FS) == 0, "overlap");
+_Static_assert(((TB_FLAG_CHERI_CAPMODE | TB_FLAG_CHERI_PCC_VALID) &
+                TB_FLAGS_MSTATUS_FS) == 0, "overlap");
 
-static inline void cpu_get_tb_cpu_state(CPURISCVState *env, target_ulong *pc,
-                                        target_ulong *cs_base, uint32_t *flags)
+#include "exec/cpu-all.h"
+#include "cpu_cheri.h"
+
+static inline void riscv_cpu_get_tb_cpu_state(CPURISCVState *env,
+                                              target_ulong *pc,
+                                              target_ulong *cs_base,
+                                              target_ulong *cs_top,
+                                              uint32_t *flags)
 {
-    *pc = PC_ADDR(env); // We want the full virtual address here and not an offset
-    *cs_base = 0;
+    *pc =
```

Comment:
Odd newline?

Reply:
Probably caused by `git-clang-format -f` because with the comment it ends up above 80 chars.